app/services/bulk_upload/lettings/row_parser.rb

```diff
@@ -167,6 +168,23 @@ def log
 
 private
 
+  def validate_relevant_collection_window
+    return unless start_date
+    return unless bulk_upload.form
+
+    unless bulk_upload.form.valid_start_date_for_form?(start_date)
+      errors.add(:field_96, I18n.t("validations.date.outside_collection_window"))
+      errors.add(:field_97, I18n.t("validations.date.outside_collection_window"))
+      errors.add(:field_98, I18n.t("validations.date.outside_collection_window"))
+    end
+  end
+
+  def start_date
+    Date.parse("20#{field_98}-#{field_97}-#{field_96}")
```

Comment:
if the year is 2009, do we know for sure that field_98 will be "09" and not "9"?

Reply:
maybe not a relevant issue until 3001 but still just wondering if we're sure!

Reply:
sure, i can pad this to be sure. yeah i have no idea why they settled for 2 digits rather than the full year

Reply:
this is padded now and test added